app/payment-link-v2/reference/get-index.controller.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
```

Comment:
This is not something that needs to be addressed in this PR, but I'm not personally super keen on the whole `get/post-index.controller` naming pattern as it makes it needlessly difficult to find files quickly as they're all named the same.

I'm more of a fan of having a single controller per page area of responsibility - e.g. `reference.controller.js` that has both the GET and POST methods in it. Maybe this is something we could discuss as a team, although shouldn't block this PR going in.

Reply:
I agree - I will raise another PR to do that as it shouldn't take too long.